Remotion/Core/Reflection.CodeGeneration.UnitTests/ClassEmitterTest.cs

```diff
@@ -103,7 +103,7 @@ public void CreateConstructorCreateField ()
           .AddStatement (new ReturnStatement());
 
       object instance = Activator.CreateInstance (classEmitter.BuildType (), "bla", 0);
-      Assert.That (instance.GetType ().GetField ("_test").GetValue (instance), Is.EqualTo ("bla"));
+      Assert.That (instance.GetType ().GetField ("_test", BindingFlags.NonPublic | BindingFlags.Instance).GetValue (instance), Is.EqualTo ("bla"));
```

Comment:
Oh great. the behavior of ClassEmitter has changed. Yes, we keep the new semantics since a) that code is not getting used in production anyway and b) re-motion can't downgrade the changes to preserve compatiblity. But, we probally have to do a breaking change documentation, at least so far as to reference the castle breaking changes.